iis-schemas/src/main/avro/eu/dnetlib/iis/importer/ProjectToOrganization.avdl

```diff
@@ -0,0 +1,10 @@
+@namespace("eu.dnetlib.iis.importer.schemas")
+protocol IIS{
+
+    record ProjectToOrganization {
```

Comment:
I don't think the name is the best one. The 'to' in the middle isn't necessary I think.